src/Page/Editor.purs

```diff
@@ -146,57 +132,84 @@ newtype EditorFields r f = EditorFields (r
   ))
 derive instance newtypeEditorFields :: Newtype (EditorFields r f) _
 
-formProxy :: F.FormProxy EditorFields
-formProxy = F.FormProxy
-
-proxies :: F.SProxies EditorFields
-proxies = F.mkSProxies formProxy
-
-validators :: forall m. Monad m => EditorFields Record (F.Validation EditorFields m)
-validators = EditorFields
-  { title: V.required >>> V.minLength 1
-  , description: V.required >>> V.minLength 1
-  , body: V.required >>> V.minLength 3
-  , tagList: F.hoistFn_ (map unwrap)
+type FormChildSlots = 
+  ( tagInput :: H.Slot (Const Void) TagInput.Message Unit )
+
+data FormAction 
+  = HandleTagInput TagInput.Message
```

Comment:
Formless can now be extended with more actions as you see fit. Here, we'll extend the component with the ability to handle messages being output by the `TagInput` component. Notice how the parent component, the `Editor` page, doesn't have to care about the tag input anymore because it's encapsulated within the form component.